http-servers/src/main/java/com/palantir/remoting/http/server/JsonExceptionMapper.java

```diff
@@ -76,6 +75,8 @@ public final Response toResponse(T exception) {
         return builder.build();
     }
 
+    abstract StatusType getStatus(T exception);
```

Comment:
This is ``StatusType`` rather than just ``Status``, because ``Response`` only gives you back ``StatusType``, which is a superclass of ``Status``